xonsh/tools.py

```diff
@@ -1582,9 +1582,13 @@ def expanduser_abs_path(inp):
 def expand_case_matching(s):
```

Comment:
No worries buddi, it's not my first time writing regexps ^__^ check my IRCBot command system for some cool regexp stuff!
https://github.com/Cheaterman/IRCBot/blob/master/commands/parser.py
Thanks for the cool tool also ^__^ appreciated!